.evergreen/config.yml

```diff
@@ -23,13 +23,6 @@ functions:
           set -o errexit
           chmod u+x libmongocrypt/.evergreen/*.sh
           ./libmongocrypt/.evergreen/print-env-info.sh
-    - command: s3.get
-      params:
-        aws_key: '${aws_key}'
-        aws_secret: '${aws_secret}'
-        remote_file: '${project}/${branch_name}/${libmongocrypt_s3_suffix}/mongo-c-driver-src.tar.gz'
-        bucket: mciuploads
-        extract_to: mongo-c-driver
```

Comment:
Every individual task now downloads its own copy of the mongo-c-driver source, no longer blocking all tasks until the `prep-c-driver-source` task.

Reply:
This is fantastic.